cmd/kube-controller-manager/app/controllermanager.go

```diff
@@ -400,6 +400,8 @@ func StartControllers(s *options.CMServer, kubeClient *client.Client, kubeconfig
 		s.ClusterName,
 		nil, nil, nil,
 		s.VolumeConfiguration.EnableDynamicProvisioning,
+		s.VolumeConfiguration.EnableNetworkStorageProvisioning,
```

Comment:
As I already wrote, **all** configuration you need should be in `StorageClass.Parameters`. Please do not introduce any new arguments.